docs/API.md

```diff
@@ -6,18 +6,19 @@
 
 ### Events
 
-#### The <td-code-editor> component has 4 events:
+#### The <td-code-editor> component has 5 events:
 
 | Name | Description |
 | --- | --- |
 | `onEditorInitialized` | Emitted when Editor is finished initializing
 | `onEditorConfigurationChanged` | Emitted when configuration of the Editor changes
 | `onEditorLanguageChanged` | Emitted when the language of the Editor changes
 | `onEditorValueChange` | Emitted when value of the Editor changes
```

Comment:
You can deprecate it, and remove it from docs without removing the implementation